src/main/java/org/prebid/server/auction/mediatypeprocessor/MediaTypeProcessor.java

```diff
@@ -0,0 +1,8 @@
+package org.prebid.server.auction.mediatypeprocessor;
+
+import com.iab.openrtb.request.BidRequest;
+
+public interface MediaTypeProcessor {
+
+    MediaTypeProcessingResult process(BidRequest bidRequest, String supportedBidderName);
```

Comment:
It would be awesome to add some documentation what the use cases are for this processor is

Reply:
Done